keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1083,6 +1105,38 @@ repeated evaluations if they are shared across validations.
     - expression: "itemMetadataNames.exists(name, name == 'required')"
 ```
 
+Since the variable names must be valid CEL names, this is checked at creation
+time.
+
+For resource constraint purposes, lazily-evaluated variables
+count towards the overall policy limit, but only once, to account for
+memoization.
+
+Informational type checking also applies to subexpressions.
```

Comment:
Please summarize how this will work. It's not obvious to me when seeing this comment.

Reply:
I've reworked this section a bit, adding a requirement that variables evaluate to some concrete type.